meetings/2024-10-22.md

```diff
@@ -1,11 +1,67 @@
-# Standard Stewardship Working Group meeting
+# Stewardship of the Standard Working Group
 
 ## 22nd October 2024
 
-Present:
+Attendees: Alex Tilley (Publish What You Fund), Bill Anderson, David Megginson (UN OCHA),  Elise Egan (USAID), Johan Zambrano (USAID), Mark Brough (Emergentally), Melina Lower (GIZ), Siem Vaessen (Zimmerman + IATI Governing Board), Sylvan Ridderinkhof (Zimmerman), Melis Karakus (Danish Refugee Council), Angelina Virchenko (UN Food and Agriculture Organization)
 
-Regrets:
+IATI Secretariat representatives: Steven Flower (Open Data Services), Bee Webb (Open Data Services), Rob Redpath (Open Data Services), Isabel Precious-Birds (Open Data Services), Cody Wallace (UNDP)
 
-### Notes
+Apologies: Amy Guy (Open Data Services), Annelise Parr (UNDP), Ryan Boles (USAID), Joshua Satre (USAID),  Marcel Laqueur (FFO Germany)
 
-### Actions
\ No newline at end of file
+Non attendees: Wakilongo Mweshwa Robert (AIGD AFRICA)
+
+### Minutes 
+
+#### Introductions and Recap
+
+* [Recap](https://docs.google.com/presentation/d/1A8YI77gWSAwnbmRglQXiqa4yum4KqpIiI1xILKkEaG0/edit?usp=sharing) of objectives of the SSWG from Steven.
+* Guiding questions have been shared in github forums.
+* Today, group is to focus on guiding principles for standard changes.
+
+#### Principles for change
+
+* Discussed proposed principles for change based on Github forum discussions.
+* The transparency of the process should also be explicit.
+* Broadening participation and outreach to affected groups is an important principle.
+* Different types of changes - use case driven, technical/maintenance driven, changes mandated by changes to other standards. 
+
+#### Principles for the Standard?
+
+* Discussed proposed principles for the standard based on Github forum discussions.
+* What is the role of the standard in the initiative and where do we want it to go?
+* Noted that there is an opportunity to bring this into the 2025-2030 Strategic Planning process.
+* Different meanings of "modularity" in the standard, and possible impact - complexity vs innovation.
+* Noted that "splitting the standard up" away from the activity model (ie: into data modules) is not currently desirable or practical.
+* Shared examples of extensions from the [Open Contracting standard](https://extensions.open-contracting.org/en/).
+* Comparability - how comparable should IATI data be from different reporting organisations?
+
+#### Responsibilities
+
+* What roles and responsibilities should the various stakeholders have in the [change process](https://iatistandard.org/en/iati-standard/upgrades/how-we-manage-the-standard/)?
+* Example provided from GitHub forum, around how the Major, Minor & Patch mechanisms already imply roles and responsibilities.
```

Comment:
@IsabelBirds I think the point here was that the existing mechanisms imply (or state) responsibilities and that they're not incorrect; just impractical or misaligned with the current setup. So, I think this sentence could end "responsibilities that provide a sound basis for future improvements", or something like that?

Reply:
@stevieflow addition

Reply:
@robredpath agree - it would be useful to link to @sylvanr's [comment](https://github.com/IATI/standard-process/discussions/2#discussioncomment-10777572), too